### PR TITLE
 Enable redis password in Java worker

### PR DESCRIPTION
**Pull request by jovany-wang**
This PR enables redis password for Java worker.

### CODE REVIEW

File: java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java [hunk 1]
```diff
@@ -146,23 +147,33 @@ public void startRayProcesses(boolean isHead) {
...
+            rayConfig.headRedisPort + i + 1, rayConfig.headRedisPassword, i);
         client.rpush("RedisShards", shard);
       }
     }
   }
 
-  private String startRedisInstance(String ip, int port, Integer shard) {
+  private String startRedisInstance(String ip, int port, String password, Integer shard) {
+    String passwordCommand = "";
+    if (!StringUtil.isNullOrEmpty(password)) {
```
Comment (jovany-wang) [inline java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java]: @raulchen  I couldn't find some thing like `Lists.of()` to replace `ImmutableLists.of()` here.
Could you plz give me some suggestions?

Comment (raulchen) [reply java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java]: You can use `Lists.newArrayList`

**Review by raulchen — COMMENTED** [review]
Linter failed, otherwise looks good

**Review by raulchen — APPROVED** [review]

**Comment by guoyuhong** [comment]
@AmplabJenkins retest this, please.

**Review by guoyuhong — APPROVED** [review]

**Pull request merged**